libc/startup/baremetal/CMakeLists.txt

```diff
@@ -1,11 +1,68 @@
+function(add_startup_object name)
```

Comment:
Can leave a TODO comment similar to https://github.com/llvm/llvm-project/blob/6752415ce8640cede492a3409af50fbef357f8b4/libc/startup/uefi/CMakeLists.txt#L1 here as well?